1.preprocessing-data/ICutils.py

```diff
@@ -0,0 +1,126 @@
+import sys
+import numpy as np
+import pathlib
+from matplotlib import pyplot as plt
+from pathlib import Path
+import os
+import skimage
+
+# explicit import to PyBaSiC due to not having package support
+sys.path.append("./PyBaSiC/")
```

Comment:
Do we need to include explicit download instructions? Oh, but you probably have these in the preprocessing README

Reply:
Yup!